pkg/cmd/registry/artifact/util/files.go

```diff
@@ -1,9 +1,13 @@
 package util
```

Comment:
I know it pre-exists this PR, but going forward we should not call packages "util", and we should update it when we see them.